GliaWidgets/CallVisualizer/VisitorCode/VisitorCodeView+NumberView.swift

```diff
@@ -0,0 +1,42 @@
+import UIKit
+
+extension CallVisualizer.VisitorCodeView {
+    class NumberView: UILabel {
+        struct Props: Equatable {
+            let character: Character
+            let style: AlertStyle
+        }
+
+        override init(frame: CGRect) {
+            super.init(frame: frame)
+            self.font = .boldSystemFont(ofSize: 40)
+            self.numberOfLines = 0
+            self.textColor = props.style.messageColor
+            self.textAlignment = .center
+            self.backgroundColor = .white
+            self.layer.backgroundColor = UIColor.white.cgColor
+            self.layer.cornerRadius = 8
+            self.layer.borderWidth = 1
+            self.layer.borderColor = UIColor(red: 0.953, green: 0.953, blue: 0.953, alpha: 1).cgColor
```

Comment:
Is there a possibility to use one of the colors that Swift provides by default? Otherwise it's not easy to know exactly which color is this.

Reply:
Unfortunately no since system grays are too dark ironically. I will make a slight change so it would be better to understand